Source/WebInspectorUI/UserInterface/Controllers/DOMDebuggerManager.js

```diff
@@ -1008,6 +1008,14 @@ WI.DOMDebuggerManager = class DOMDebuggerManager extends WI.Object
             }
         }
     }
+
+    // FIXME <https://webkit.org/b/298981> Add DOMDebugger support for FrameTarget.
+    *_allSupportedTargets() {
```

Comment:
Ooh, cool new generator syntax for me to figure out :P

Though I suppose fully modern syntax would be 

```
*#allSupportedTargets() {
```